index.html

```diff
@@ -6,21 +6,30 @@
   </head>
   <body>
     <h1>Welcome to the Salmon Cookies Website</h1>
-    <img src="https://raw.githubusercontent.com/codefellows/seattle-201d19/c22e563a40184ee90e8a1cfb930f6aa7e65b3571/class-06-dom-objects-domain_modeling/assets/salmon.png">
-      <div id = "leftmenu">
+    <img src="C:\Users\Isak Swearingen\Pictures\Saved Pictures\salmon.png">
+    <div class="aboutus">
+      <h2>About Us</h2>
+      <p>Some Stuff</p>
+    </div>
+    <div class = 'locations'>
```

Comment:
Use double quotes in html, and single quotes in your javascript. Also, don't put spaces around equals signs in html.